samples/pt-br/Primeiros Passos/index.html

```diff
@@ -26,7 +26,7 @@
     -->
         
     <p>
-        Brackets é um editor diferente e tem algumas características únicas, como a Edição Rápida (Quick Edit), Visualização ao vivo (Live Preview) e muitas outras que você não encontrará em outros editores. E o Brackets é escrito em JavaScript, HTML e CSS. Isso significa que a maioria de vocês que utilizam o Brackets possuem as habilidades necessárias para modificar e estender o editor. Na verdade, usamos o Brackets todos os dias para desenvolver o Brackets. Para saber mais sobre como utilizar os principais recursos, continue lendo.
+        O Brackets é um editor diferente e tem algumas características únicas, como a Edição Rápida (Quick Edit), Visualização ao vivo (Live Preview) e muitas outras que você não encontrará em outros editores. E o Brackets é escrito em JavaScript, HTML e CSS. Isso significa que a maioria de vocês que utilizam o Brackets possuem as habilidades necessárias para modificar e estender o editor. Na verdade, usamos o Brackets todos os dias para desenvolver o Brackets. Para saber mais sobre como utilizar os principais recursos, continue lendo.
```

Comment:
Like @brunnopleffken commented earlier, the original page says:

"<em>Brackets is a different type of editor.</em>
 Brackets has some unique features like Quick Edit, [...]"

Maybe you could maintain the same structure with something like:

"<em>O Brackets é um editor diferente.</em>
O Brackets tem algumas características únicas, como a Edição Rápida [...]"